examples/runtimeDashboard/output.json

```diff
@@ -232,7 +232,7 @@
             "h": 8,
             "w": 8,
             "x": 0,
-            "y": 10
+            "y": 9
```

Comment:
If this is a refactor, how come your output has changed?

Reply:
This happens in commit 3, before the refactoring. Commit 2 fixes a few bugs in makeGrid, these fixes change the output.

Only commit 6 is a change of the output that I didn't fix in commit 2, I'm being pragmatic here.